README.md

```diff
@@ -1,7 +1,27 @@
 klog
 ====
 
-klog is a permanant fork of https://github.com/golang/glog. original README from glog is below
+klog is a permanent fork of https://github.com/golang/glog.
+
+## Why was klog created?
+
+The decision to create klog was one that wasn't made lightly, but it was necessary due to some
+drawbacks that are present in [glog](https://github.com/golang/glog). Ultimately, the fork was created due to glog not being under active development; this can be seen in the glog README:
+
+> The code in this repo [...] is not itself under development
+
+This makes us unable to solve many use cases without a fork. The factors that contributed to needing feature development are listed below:
+
+ * `glog` presents a lot "gotchas" that aren't well documented, not necessarily needed (specifically challenges that are present in container environments). This makes using `glog` in shared libraries somewhat dangerous and unpredictable 
```

Comment:
I'm not sure what "not necessarily needed"  refers to here?

Reply:
Yeah, honestly not sure what happened there. Will fix.